catalog/tests/PartitionScheme_unittest.cpp

```diff
@@ -343,15 +363,43 @@ TEST(PartitionSchemeHeaderTest, VarCharRangePartitionSchemeHeaderTest) {
   for (size_t i = 0; i < num_strings; ++i) {
     EXPECT_EQ(kExpectedPartitions[i],
               partition_scheme_header->getPartitionId(
-                  TypedValue(kVarChar,
-                             kSampleStrings[i],
-                             std::strlen(kSampleStrings[i]) + 1)));
+                  { TypedValue(kVarChar, kSampleStrings[i], std::strlen(kSampleStrings[i]) + 1) }));
   }
 }
 
+TEST(PartitionSchemeHeaderTest, MultiAttributeRangePartitionSchemeHeaderTest) {
+  vector<PartitionSchemeHeader::PartitionValues> partition_ranges;
+  // Partition boundaries are { 0, 0.00000 }, { 10, 10.0000 }, { 20, 20.00000 }
+  // Last partition can hold upto infinity.
+  // First partition can hold from { -infinity, -infinity } to { 0, -2^(-1074) }.
```

Comment:
Can you explain -2^(-1074)? I thought the first partition should be < {0, 0.0000} and > {-Inf, -Inf}.